tango_simlib/tango_launcher.py

```diff
@@ -31,37 +31,42 @@
 required_argument('--server-command', help="TANGO server executable command")
 required_argument('--server-instance', help="TANGO server instance name")
 required_argument('--port', help="TCP port where TANGO server should listen")
+parser.add_argument('--file', help="ASCII file containing device configuration parameters" +
+                    "and device network access parameter")
 parser.add_argument('--put-device-property', action='append', help=
                     "Put a device property into the TANGO DB, format is: "
                     "'device/name/X:property_name:property_value'. Only allows "
                     "properties to be set on devices started with this command. "
                     "Can be specified multiple times.",
                     dest='device_properties', default=[])
 
-def register_device(name, device_class, server_name, instance):
-    dev_info = tango.DbDevInfo()
-    dev_info.name = name
-    dev_info._class = device_class
-    dev_info.server = "{}/{}".format(server_name.split('.')[0], instance)
-    print """Attempting to register TANGO device {!r}
-    class: {!r}  server: {!r}.""".format(
+def register_device(name, device_class, server_name, instance, file=None):
+    if file:
+        db = tango.Database(file)
```

Comment:
You are creating an instance of the `Database` with a `file` parameter but not using it in this function.